fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -11918,7 +11968,7 @@
 | `value=(string 'false')`| (changed)| `"false"` |
 | `value=(string 'on')`| (changed)| `"on"` |
 | `value=(string 'off')`| (changed)| `"off"` |
-| `value=(symbol)`| (initial, warning)| `<empty string>` |
+| `value=(symbol)`| (initial, warning, ssr error, ssr mismatch)| `<empty string>` |
```

Comment:
Warning on client and server in React 17: https://codesandbox.io/s/react-17-option-value-symbol-cy2rf
Throws on server in React 18 but also has an additional warning: https://codesandbox.io/s/react-18-option-value-symbol-forked-kbc2n

Reply:
Interesting. Adding to TODOs to ensure this is intentional.